index.js

```diff
@@ -1,44 +1,39 @@
+const os = require('os');
 const fs = require('fs');
 const path = require('path');
+const execFile = require('child_process').execFile;
 
-const exec = require('child_process').exec;
-const nodePath = process.execPath;
-const version = process.versions.v8;
-const tmpfile = path.join(__dirname, version+'.flags.json');
+const tmpfile = path.join(os.tmpdir(), process.versions.v8+'.flags.json');
 const exclusions = ['--help'];
 
 module.exports = function (cb) {
 
-  if (fs.existsSync(tmpfile)) {
-    process.nextTick(function () {
-      cb(null, require(tmpfile))
-    });
-  } else {
-    exec('"'+nodePath+'" --v8-options', function (execErr, result) {
+  fs.exists(tmpfile, function (exists) {
+    if (exists) {
+      return cb(null, require(tmpfile))
```

Comment:
This is the most common mistake with `fs.exists`. It's susceptible to a race condition as the file might change in between `fs.exists` and `require`. Nothing will probably happen, but it could.

You should rather read it directly and check for and handle the `ENOENT` error. Or in this case try/catch the require.